tests/foreman/cli/test_capsulecontent.py

```diff
@@ -0,0 +1,206 @@
+    target_sat.cli.Capsule.content_synchronize(
+        {'id': module_capsule_configured.nailgun_capsule.id, 'organization-id': function_org.id}
+    )
+    target_sat.wait_for_tasks(
+        search_query='label = Actions::Katello::CapsuleContent::UpdateContentCounts',
+        search_rate=5,
+        max_tries=5,
+    )
+
+    # Get the content counts from Satellite side and compare them with Capsule.
+    info = target_sat.cli.Capsule.content_info(
+        {'id': module_capsule_configured.nailgun_capsule.id, 'organization-id': function_org.id}
+    )
+
+    caps_counts = [
+        repo
+        for repo in info['lifecycle-environments']['1']['content-views']['1'][
+            'repositories'
+        ].values()
+    ]
+    sat_repos = [
+        target_sat.cli.Repository.info({'id': repo['repository-id']}) for repo in caps_counts
+    ]
+
+    for s_repo in sat_repos:
+        c_repo = next(r for r in caps_counts if r['repository-id'] == s_repo['id'])
+        common_keys = set(s_repo['content-counts'].keys()) & set(c_repo['content-counts'].keys())
+        assert len(common_keys), f'''No common keys found for type "{s_repo['content-type']}".'''
+        assert expected_keys[s_repo['content-type']].issubset(common_keys), (
+            'Some fields are missing: expected '
+            f'''{expected_keys[s_repo['content-type']]} but found {common_keys}'''
+        )
+        assert all(
+            [
+                s_repo['content-counts'].get(key) == c_repo['content-counts'].get(key)
+                for key in common_keys
+            ]
+        )
+
+    # Remove the LCE from Capsule and ensure it's not listed.
+    res = target_sat.cli.Capsule.content_remove_lifecycle_environment(
+        {
+            'id': module_capsule_configured.nailgun_capsule.id,
+            'organization-id': function_org.id,
+            'environment-id': function_lce.id,
+        }
+    )
+    assert 'environment successfully removed' in str(res)
+
+    info = target_sat.cli.Capsule.content_info(
+        {'id': module_capsule_configured.nailgun_capsule.id, 'organization-id': function_org.id}
+    )
+    assert len(info['lifecycle-environments']) == 0, 'The LCE is still listed'
+
+
+@pytest.mark.stream
+def test_positive_update_counts(target_sat, module_capsule_configured):
+    """Verify the update counts functionality
+
+    :id: 5658dbb1-3d3d-4926-804b-1ff221cf5075
+
+    :setup:
+        1. Satellite with registered Capsule.
+
+    :steps:
+        1. Run capsule content update-counts via hammer.
+
+    :expectedresults:
+        1. Update Content Counts task is created and succeeds.
+
+    """
+    task = target_sat.cli.Capsule.content_update_counts(
+        {'id': module_capsule_configured.nailgun_capsule.id, 'async': True}
+    )
+    target_sat.wait_for_tasks(
+        search_query='label = Actions::Katello::CapsuleContent::UpdateContentCounts'
+        f' and id = {task["id"]}',
+        search_rate=5,
+        max_tries=5,
+    )
```

Comment:
Don't we assert anything? Is it enough that the task is marked as finished?

Reply:
I think so, here we just assert that the task can be triggered via CLI and succeeds (default `must_succeed=True`).
What the task did or did not is asserted in the previous (and API) tests because this task is executed after each capsule sync (see L130 above)